src/home_robot/home_robot/manipulation/heuristic_pick_policy.py

```diff
@@ -237,6 +331,54 @@ def get_action(
             action = DiscreteNavigationAction.STOP
         return action, vis_inputs
 
+    def _find_grasp_cfg(self, pt, verbose=True):
+        """Find grasp config for the arm at pt = (x, y, z)"""
+        # Top down grasp
+        pos_top = pt.copy()
+        pos_top[2] += STRETCH_STANDOFF_DISTANCE
+        R = tra.euler_matrix(0, np.pi, 0)
+        rot_top = tra.quaternion_from_matrix(R)
+        # Side grasp
+        pos_side = pt.copy()
+        pos_side[1] += STRETCH_STANDOFF_DISTANCE
+        rot_side = tra.quaternion_from_euler(np.pi / 2, 0, 0)
+
+        model = HelloStretchKinematics()
+
+        if verbose:
+            print(
+                "Raising the point by the size of the stretch gripper before doing IK..."
+            )
+            print("[TOP] Target point to grasp:", pos_top)
+            print("[SIDE] Target point to grasp:", pos_side)
+
+        cfg, res, info = model.manip_ik((pos_top, rot_top))
```

Comment:
Do we not need to pass `q0=STRETCH_PREGRASP_Q` here. Robot joints are in `STRETC_PREGRASP_Q` after manipulation mode. Also, there's a default `STRETCH_HOME_Q` being used inside `manip_ik`

Reply:
We do not need to, but this can help, i will add it